tools/releaseBuild/build.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Windows": {
+        "Name": "win7-x64",
+        "RepoDestinationPath": "C:\\PSScriptAnalyzer",
+        "BuildCommand": "C:\\containerFiles\\buildPSSA.ps1",

```

Comment:
non-blocking:
`_DockerVolume_` is supposed to be passed as a parameter for where you are supposed to copy the results to.  

This will become more important when we migrate to a new build system and the path is not the same